core/lens/lens_list.ml

```diff
@@ -0,0 +1,42 @@
+open Utility
+
```

Comment:
Similar list operators are available in the `Utility`, and those that arent', you could just implement in `ListUtils`. I think this module is unnecessary altogether.

Reply:
As part of reducing the dependency on Links, I have decided to make my code more compatible with the OCaml `Base` library rather than on the somewhat chaotic `Utility` module. If the lens code is ever seperated into its own library, it won't need the `Utility` module and instead all the `lens_*` utility files can be deleted and replaced with a `Base` library reference.

Reply:
It may be appropriate to consider moving over to use Jane Street's `Base` library at some point - perhaps we can discuss it at our next developer meeting. But it is probably not appropriate for a branch to make this kind of change independently of the trunk - at least if you want the branch to be merged into the trunk.

Reply:
As it stands the code doesn't change any of the other links code, it doesn't include any further libraries and any of the further modules are "contained" in the lens folder under a separate namespace. I don't see the problem.